dm/config/task_test.go

```diff
@@ -429,7 +440,7 @@ func (t *testConfig) TestGenAndFromSubTaskConfigs(c *C) {
 				Security:         &security,
 				RawDBCfg:         &rawDBCfg,
 			},
-			RouteRules:  []*router.TableRule{&routeRule1, &routeRule2},
+			RouteRules:  []*router.TableRule{&routeRule2, &routeRule1},
```

Comment:
we could test different subtask has different route rules, and merged task won't lost any of them for new implementation of `FromSubTaskConfigs `